docs/en/reference/working-with-associations.rst

```diff
@@ -500,6 +500,18 @@ and the "remove" operation.
         //...
     }
 
+Now you can persist the `User` entity like that:
```

Comment:
Instead of `Now`, describe the context again. Something like `Since cascade persist is configured for the User#commentsAuthored association, you can now create a user and persist its comments with it as following:`